### ISSUE TITLE
在阿里小程序引入会报错

### ISSUE THREAD

**Issue by SevenGuns** [antvis/vis-predict-engine#6]
node_modules/@tensorflow/tfjs-layers/dist/callbacks.js: Unexpected token (31:7)
node_modules/@tensorflow/tfjs-data/dist/datasets/csv_dataset.js: Unexpected token (56:7)
node_modules/@tensorflow/tfjs-layers/dist/models.js: Unexpected token (66:7)
node_modules/@tensorflow/tfjs-data/dist/sources/url_data_source.js: Unexpected token (41:7)
node_modules/@tensorflow/tfjs-layers/dist/engine/topology.js: Unexpected token (47:7)
node_modules/@tensorflow/tfjs-core/dist/tensor.js: Unexpected token (44:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/adadelta_optimizer.js: Unexpected token (46:7)
node_modules/@tensorflow/tfjs-layers/dist/variables.js: Unexpected token (38:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/momentum_optimizer.js: Unexpected token (42:7)
node_modules/@tensorflow/tfjs-core/dist/environment.js: Unexpected token (45:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/recurrent.js: Unexpected token (232:7)
node_modules/@tensorflow/tfjs-layers/dist/engine/training.js: Unexpected token (381:7)
node_modules/@tensorflow/tfjs-layers/dist/base_callbacks.js: Unexpected token (59:7)
node_modules/@tensorflow/tfjs-data/dist/sources/file_data_source.js: Unexpected token (42:7)
node_modules/@tensorflow/tfjs-converter/dist/executor/graph_model.js: Unexpected token (49:7)
node_modules/@tensorflow/tfjs-layers/dist/engine/input_layer.js: Unexpected token (30:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/convolutional_recurrent.js: The keyword 'class' is reserved (50:0)
node_modules/@tensorflow/tfjs-data/dist/readers.js: Unexpected token (149:39)
node_modules/@tensorflow/tfjs-core/dist/platforms/platform_browser.js: Unexpected token (38:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/noise.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-data/dist/dataset.js: Unexpected token (67:7)
node_modules/@tensorflow/tfjs-core/dist/platforms/platform_node.js: Unexpected token (53:7)
node_modules/@tensorflow/tfjs-core/dist/serialization.js: Unexpected token (40:7)
node_modules/@tensorflow/tfjs-core/dist/util.js: Identifier 'fetch' has already been declared (107:16)
node_modules/@tensorflow/tfjs-core/dist/optimizers/adamax_optimizer.js: Unexpected token (45:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/adam_optimizer.js: Unexpected token (48:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/sgd_optimizer.js: Unexpected token (41:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/wrappers.js: Unexpected token (49:7)
node_modules/@tensorflow/tfjs-data/dist/datasets/text_line_dataset.js: Unexpected token (40:7)
node_modules/@tensorflow/tfjs-core/dist/engine.js: The keyword 'class' is reserved (46:0)
node_modules/@tensorflow/tfjs-core/dist/ops/browser.js: Unexpected token (169:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/rmsprop_optimizer.js: Unexpected token (47:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/adagrad_optimizer.js: Unexpected token (44:7)
node_modules/@tensorflow/tfjs-core/dist/io/indexed_db.js: Unexpected token (50:7)
node_modules/@tensorflow/tfjs-layers/dist/constraints.js: Unexpected token (48:7)
node_modules/@tensorflow/tfjs-core/dist/optimizers/optimizer.js: Unexpected token (38:7)
node_modules/@tensorflow/tfjs-core/dist/backends/backend.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-core/dist/io/local_storage.js: Unexpected token (100:7)
node_modules/@tensorflow/tfjs-core/dist/io/browser_files.js: Unexpected token (51:7)
node_modules/@tensorflow/tfjs-layers/dist/initializers.js: Unexpected token (45:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/convolutional_depthwise.js: Unexpected token (78:7)
node_modules/@tensorflow/tfjs-core/dist/io/http.js: Unexpected token (49:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/advanced_activations.js: Unexpected token (38:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/core.js: Unexpected token (42:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/merge.js: Unexpected token (43:7)
node_modules/@tensorflow/tfjs-core/dist/io/passthrough.js: The keyword 'class' is reserved (33:0)
node_modules/@tensorflow/tfjs-core/dist/io/router_registry.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-core/dist/io/model_management.js: Unexpected token (46:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/convolutional.js: Unexpected token (274:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/embeddings.js: Unexpected token (42:7)
node_modules/@tensorflow/tfjs-core/dist/ops/setdiff1d_async.js: Unexpected token (65:6)
node_modules/@tensorflow/tfjs-core/dist/optimizers/optimizer_constructors.js: Unexpected token (40:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/padding.js: Unexpected token (96:7)
node_modules/@tensorflow/tfjs-backend-cpu/dist/backend_cpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-core/dist/ops/image/non_max_suppression_padded_async.js: Unexpected token (63:6)
node_modules/@tensorflow/tfjs-backend-webgl/dist/unaryop_packed_gpu.js: Unexpected token (37:7)
node_modules/@tensorflow/tfjs-layers/dist/errors.js: Unexpected token (38:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/clip_packed_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-core/dist/ops/boolean_mask.js: Unexpected token (57:6)
node_modules/@tensorflow/tfjs-layers/dist/layers/normalization.js: Unexpected token (174:7)
node_modules/@tensorflow/tfjs-core/dist/ops/image/non_max_suppression_with_score_async.js: Unexpected token (66:6)
node_modules/@tensorflow/tfjs-layers/dist/regularizers.js: Unexpected token (39:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/depth_to_space_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/diag_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/dilation_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/fill_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/gather_nd_gpu.js: Unexpected token (18:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/gather_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/lrn_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-layers/dist/layers/pooling.js: Unexpected token (136:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/backend_webgl.js: Unexpected token (84:7)
node_modules/@tensorflow/tfjs-core/dist/ops/where_async.js: Unexpected token (55:6)
node_modules/@tensorflow/tfjs-core/dist/ops/in_top_k.js: Unexpected token (53:6)
node_modules/@tensorflow/tfjs-core/dist/io/io_utils.js: Unexpected token (56:7)
node_modules/@tensorflow/tfjs-core/dist/ops/image/non_max_suppression_async.js: Unexpected token (57:6)
node_modules/@tensorflow/tfjs-backend-webgl/dist/mulmat_packed_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/im2col_packed_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/argminmax_packed_gpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/unaryop_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/pool_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/gpgpu_context.js: Unexpected token (40:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/addn_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/reduce_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/argminmax_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/mean_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/tile_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/avg_pool_backprop_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/strided_slice_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/addn_packed_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/batchnorm_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-core/dist/io/weights_loader.js: Unexpected token (49:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/complex_abs_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/clip_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/binaryop_gpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/batchnorm_packed_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/crop_and_resize_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/conv_gpu_depthwise.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/binaryop_packed_gpu.js: Unexpected token (39:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/kernels/FromPixels_utils/from_pixels_packed_gpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/scatter_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/slice_packed_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/lrn_packed_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/slice_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/fft_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/conv_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/conv_backprop_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/cumsum_gpu.js: Unexpected token (18:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/reverse_packed_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/reverse_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/select_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/rotate_gpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/resize_bilinear_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/reshape_packed_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/concat_packed_gpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/concat_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-core/dist/ops/rand_util.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/mirror_pad_packed_gpu.js: Unexpected token (74:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/resize_nearest_neighbor_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/resize_nearest_neighbor_backprop_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/resize_bilinear_packed_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/multinomial_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/transpose_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/pad_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/conv_packed_gpu_depthwise.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/conv_backprop_gpu_depthwise.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/flip_left_right_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/pad_packed_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/mirror_pad_gpu.js: Unexpected token (34:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/binaryop_complex_gpu.js: Unexpected token (42:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/onehot_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/kernels/FromPixels_utils/from_pixels_gpu.js: Unexpected token (36:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/segment_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/max_pool_backprop_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/lrn_grad_gpu.js: Unexpected token (33:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/resize_bilinear_backprop_gpu.js: Unexpected token (35:7)
node_modules/@tensorflow/tfjs-backend-webgl/dist/transpose_packed_gpu.js: Unexpected token (35:7)
Please set enableNodeModuleBabelTransform to true or add "@tensorflow/tfjs-backend-cpu, @tensorflow/tfjs-backend-webgl, @tensorflow/tfjs-converter, @tensorflow/tfjs-core, @tensorflow/tfjs-data, @tensorflow/tfjs-layers" to node_modules_es6_whitelist in mini.project.json for node_modules babel transform
